src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -71,6 +72,8 @@
   private final JestClient client;
   private final Version version;
 
+  private final Map<String, Boolean> indexCache = new ConcurrentHashMap<>();
```

Comment:
since this is local to each task and read and updated in a single method, we could get away with just `HashMap<>()`?

Reply:
fair point, I somehow put the concurrent behviour as there is the public method:

```java
  public void start(Map<String, String> props, ElasticsearchClient client) {
```

apologies for that.